src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * Responsible for writing and loading the list of tasks to and from the user's memory.
+ */
+public class Storage {
+
+    static final String PROJECT_ROOT = System.getProperty("user.dir");
```

Comment:
Should there be an access modifier here?

There seems to be a missing access modifier.

Considering adding a private statement here.